utils.sh

```diff
@@ -235,6 +235,7 @@ function sync_repo_and_patch {
 }
 
 function generate_auth_template {
+    set +x
```

Comment:
wondering if this can be made optional in case we need it for troubleshooting
also maybe a message informing what we're doing could be useful